### PR TITLE
[MIG] 14.0 algolia

### PR DESCRIPTION
**Pull request by PierrickBrun**
based on #855 

connector_algolia will need to be merged in 14.0 before merging this

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by kevinkhao** [comment]
dependencies seem OK @Cedric-Pigeon

**Comment by rousseldenis** [comment]
> dependencies seem OK @Cedric-Pigeon

Still needs connector_algolia

**Comment by rousseldenis** [comment]
> > dependencies seem OK @Cedric-Pigeon
> 
> Still needs connector_algolia

@PierrickBrun I think you did not create PR for your 14.0-shopinvader there https://github.com/OCA/search-engine/pulls

**Comment by PierrickBrun** [comment]
> > > dependencies seem OK @Cedric-Pigeon
> > 
> > 
> > Still needs connector_algolia
> 
> @PierrickBrun I think you did not create PR for your 14.0-shopinvader there https://github.com/OCA/search-engine/pulls

I will update https://github.com/OCA/search-engine/pull/67 when https://github.com/OCA/search-engine/pull/73 is merged. I created 14.0-shopinvader to be able to run the CI, but did not intend to merge it

**Review by mourad-ehm — APPROVED** [review]

**Comment by simahawk** [comment]
NOTE: to be rebased and test against https://github.com/OCA/search-engine/pull/86 (needs review)
All other PRs have been merged.

**Review by simahawk — CHANGES_REQUESTED** [review]
just to prevent accidental merge ;)

**Comment by simahawk** [comment]
@PierrickBrun can you rebase/cleanup/etc?

**Comment by PierrickBrun** [comment]
> @PierrickBrun can you rebase/cleanup/etc?

@simahawk  done

File: shopinvader_algolia/__manifest__.py [hunk 1]
```diff
@@ -18,6 +18,6 @@
     ],
     "demo": ["demo/backend_demo.xml", "demo/product_demo.xml"],
     "external_dependencies": {"python": ["vcrpy_unittest"]},
```
Comment (simahawk) [inline shopinvader_algolia/__manifest__.py]: this a dep only for tests, can you remove it?

**Review by simahawk — APPROVED** [review]
@PierrickBrun one small thing to change: move test dep to test-dependencies.txt

**Review by sebastienbeau — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**